### ISSUE TITLE
Not everything is under DST_DIR

### ISSUE THREAD

**Issue by Kerndog73** [AlexanderWillner/runMacOSinVirtualBox#101]
**To Reproduce**
```
export DST_DIR="somewhere_other_than_home/VirtualBox VMs"
make vm
ls ~/VirtualBox\ VMs
# There's a macOS-VM folder in here
```

**Expected behavior**
I expect everything to be under `somewhere_other_than_home/VirtualBox VMs` but there are still some files in `~/VirtualBox VMs`. The script doesn't seem to be aware of these files. `make clean` and `make stash` don't delete the files under `~/VirtualBox VMs`.

**Additional context**
I ran into this because I was trying to recreate the VM. I deleted everything under `somewhere_other_than_home/VirtualBox VMs` but then I was running into errors. I inspected the log realised the error was caused by the presence of the some files in `~/VirtualBox VMs`.

**Comment by roamingthings** [comment]
Is this addressed by #97 ?

**Comment by Kerndog73** [comment]
@roamingthings That seems to have solved it. I'm not sure about `make stash` though. I don't want to test it because I've started over too many times today!

**Comment by AlexanderWillner** [comment]
Thanks, merged #97
